docs/getting-started/installation.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Installation
+description: Installing Powertools for AWS Lambda
+---
+
+<!-- markdownlint-disable MD043 -->
+
+You can use Powertools for AWS Lambda (TypeScript) by installing it with your favorite dependency management, or via [Lambda Layers](./lambda-layers.md).
+
+The toolkit is compatible with both TypeScript and JavaScript code bases, and supports both CommonJS and ES modules.
+
+All features are available as individual packages, so you can install only the ones you need, for example:
+
+* **Logger**: `npm i @aws-lambda-powertools/logger`{.copyMe}:clipboard:
```

Comment:
Maybe this is not possible easily but I tried to click the copy icon to copy the text which didn't work.

Reply:
I think it's browser specific - I noticed the same on Safari but it works on Chrome.

I'll take a look in a separate issue/PR since it'll require to modify the JS script code, and I want to keep this PR focused.

Reply:
Agreed. I am using Firefox by the way.

Reply:
Created an issue to track this #3832.